test/config.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import { shallow } from "enzyme";
+import * as React from "react";
+
+describe("Config variables", () => {
+  let savedEnvs: any;
+
+  beforeEach(() => {
+    savedEnvs = process.env;
+    process.env = {};
+  });
+
+  afterEach(() => {
+    process.env = savedEnvs;
+  });
+
+  it("Should load variables", () => {
+    process.env = {
+      COMMITMENT_CONTRACT_ADDRESS: "0x00000000000000000000000000000000000000000",
+    };
+    const { commitmentContractAdress } = require("../app/config");
+    expect(commitmentContractAdress).to.equal(process.env.COMMITMENT_CONTRACT_ADDRESS);
+  });
+
+  it('Should throw error that "Key" is not exists', () => {
+    process.env = {};
+    expect(() => {
+      delete require.cache[require.resolve("../app/config")];
```

Comment:
why not use one of these clear require modules? Problem with this test is that if you would run tests in different order they won't pass. Please move it to beforeEach or use external module to deal with it.